### PR TITLE
chore: Enable dredd tests for some of the Orders API endpoints

### PR DESCRIPTION
**Pull request by blossom2017**
#7014 

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/custom/orders.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def create_order():
     db.session.commit()
     order.populate_and_save()
 
-    return OrderSchema().dumps(order)
+    return OrderSchema().dump(order)
```
Comment (iamareebjamal) [inline app/api/custom/orders.py:170]: Why dump instead of dumps?

File: docs/api/blueprint/order/orders.apib [hunk 2]
```diff
@@ -793,3 +626,114 @@ Create paypal payment
             "payment_id": "example",
             "error": null
         }
+
+## Create Order [/v1/orders/create-order]
```
Comment (iamareebjamal) [inline docs/api/blueprint/order/orders.apib:630]: This should be placed in create order (1st section) place

File: tests/hook_main.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import os.path as path
 import sys
+import json
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply tests/hook_main.py:3]: This is unused

File: app/api/custom/orders.py (hunk 1 above)
Comment (blossom2017) [reply app/api/custom/orders.py:170]: dumps was returning a string -> content/html whereas dump returned dictionary application/json. I saw that in the other places dump had been used.

File: docs/api/blueprint/order/orders.apib (hunk 2 above)
Comment (blossom2017) [reply docs/api/blueprint/order/orders.apib]: Done.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**